tests/integration/data/source/database/adapter/MySqlTest.php

```diff
@@ -353,6 +353,18 @@ public function testDefaultValues() {
 		$this->assertPattern('$\d{4}-\d\d-\d\d \d\d:\d\d:\d\d$', $result['created']);
 		$this->assertTrue(time() - strtotime($result['created']) < 24 * 3600);
 	}
+
+	/**
+	 * Verifies that setting options using a raw SQL string works, when
+	 * the operation returns no result.
+	 *
+	 * @link https://github.com/UnionOfRAD/lithium/issues/1210
+	 */
+	public function testRawOptionSettingWithNoResultResource() {
+		$expected = array();
+		$result = Galleries::connection()->read('SET SESSION group_concat_max_len = 100000;');
```

Comment:
As the query is actually executed, following tests might be affected by the change. Is there any kind of query that fulfills our test purposes but does not change the server/client behavior?